src/main/java/Duke.java

```diff
@@ -6,5 +17,234 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        System.out.print("____________________\n"+
+                          "Hello! I'm Duke\n"+
+                          "What can I do for you?\n");
+        boolean isToQuit = false;
+        String input;
+
+        while(!isToQuit){
+            input = sc.nextLine();
+            String commandWord = getCommand(input);
+            String info = getInfo(input);
+
+            switch(commandWord){
+            case "todo":
+                try{
+                    addTodo(info, tasks);
+                }catch (DukeEmptyStringException e){
+                    System.out.println("Task cannot be empty");
+                }
+                break;
+            case "deadline":
+                try{
+                    addDeadline(info, tasks);
+                }catch (DukeEmptyStringException e){
+                    System.out.println("Task cannot be empty");
+                }catch (DukeInsufficientInfoException f){
+                    System.out.println("Deadline should have /by field");
+                }
+                break;
+            case "event":
+                try{
```

Comment:
Consider extracting magic strings such as "deadline" and "event" into constants.